drivers/display/display_ili9340.c

```diff
@@ -8,37 +8,31 @@
 #include "display_ili9340.h"
 #include <drivers/display.h>
 
-#define LOG_LEVEL CONFIG_DISPLAY_LOG_LEVEL
 #include <logging/log.h>
-LOG_MODULE_REGISTER(display_ili9340);
+LOG_MODULE_REGISTER(display_ili9340, CONFIG_DISPLAY_LOG_LEVEL);
 
 #include <drivers/gpio.h>
 #include <sys/byteorder.h>
 #include <drivers/spi.h>
 #include <string.h>
 
+#define RES_X_MAX 320U
```

Comment:
This change should be in own commit and use DT height and width properties.

Reply:
This is not related to DT config. This high bound for the ili9xxx drivers.